utilities/src/biginteger/bigint_384.rs

```diff
@@ -304,7 +304,17 @@ impl Debug for BigInteger384 {
 }
 impl Display for BigInteger384 {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(f, "{}", self.to_biguint())
+        #[cfg(not(feature = "cosmwasm"))]
+        {
+            write!(f, "{}", self.to_biguint())
+        }
+
+        #[cfg(feature = "cosmwasm")]
+        {
+            let bytes = self.to_bytes_le().unwrap().try_into().unwrap();
+            let num = cosmwasm_std::Uint256::new(bytes);
+            write!(f, "{}", num)
```

Comment:
I believe it would be better to return a `std::fmt::Result::Error` here  here instead of unwrapping.